source/controller/project_controller.py

```diff
@@ -409,18 +412,17 @@ def handle_tree_double_click(self):
         else:
             pass
         self.visualize_graph_dock.setDisabled(False)
+        print(project_information_store.get_file_name())
+        self.project_window.set_title_bar(project_information_store.get_file_name())
 
-    def update_graph_to_table(self, edited_graph):
-        self.edited_graph = edited_graph
-        self.visualize_graph_dock.current_graph = self.edited_graph
+    def update_graph_to_table(self):
+        graph = project_information_store.current_graph
 
         for key in self.invariants_selected.keys():
-            if len(self.edited_graph.nodes) == 0:
-                self.invariants_selected[key] = \
-                    "Null Graph"
+            if len(graph) == 0:
+                self.invariants_selected[key] = "Null Graph"
```

Comment:
Change this message to "None graph", because "Null graph" is another situation.

Reply:
Done!